### PR TITLE
Feat/tv2 contributions

### PR DESCRIPTION
**Pull request by nytamin**
This PR contains various things from the olzzon fork that for some reason hasnt made it upstream yet.

A summary of the changes:

* Now subscribes to expectedPlayoutItems, to support preloading of vizMSE graphics.
* piping of events introduced in https://github.com/nrkno/tv-automation-state-timeline-resolver/pull/176

Note: TSR needs to be updated before merging this.

### CODE REVIEW

**Review by sparkpunkd — APPROVED** [review]
Great to get these features in core Sofie 😃

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
 		"unlinkall": "yarn unlink @sofie-automation/server-core-integration timeline-state-resolver && yarn --check-files",
 		"validate:dependencies": "yarn audit --groups dependencies && yarn license-validate",
 		"validate:dev-dependencies": "yarn audit --groups devDependencies",
-		"license-validate": "yarn sofie-licensecheck --allowPackages cycle@1.0.3",
+		"license-validate": "node-license-validator -p -d --allow-licenses MIT 0BSD BSD BSD-2-Clause BSD-3-Clause ISC Apache Apache-2.0 Unlicense WTFPL --allow-packages cycle",
```
Comment (Julusian) [inline package.json:45]: all these license checker changes arent wanted. If they are necessary then we should update the preset for the new licenses

Comment (nytamin) [reply package.json]: I actually fixed that this morning, but forgot to push.. :)

**Comment by baltedewit** [comment]
Closing this as it is replaced by https://github.com/nrkno/tv-automation-server-core/pull/501

**Pull request closed without merging**